decentraland/unity-explorer
# chore: scene reloading through debug menu / sdk6 label

## What does this PR change?

- Adds a button to reload SDK7 scenes
- Adds a label to show if we are on an sdk6 scene
![image](https://github.com/decentraland/unity-explorer/assets/1999557/8b117107-3e89-42b7-9760-0712a9374172)


## How to test the changes?

1. Launch the explorer
2. Go to a SDK7 scene
3. Open the debug menu through /debug
4. Go to the Scene Reload tab and press the reload button

For the label
1. If you start in Genesis Plaza (SDK7), the label should not be there
2. If you go to the roads or any sdk6 scene (just the lod) the label should be there
3. If you got to an empty parcel, the label should not be there
4. If you got back to an SDK7 scene, the label should be there
Teleport between locations and worlds to confirm the label usage

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
